src/ekka_rlog_schema.erl

```diff
@@ -87,12 +87,16 @@ init(boot) ->
                                                      {record_name, ?schema},
                                                      {attributes, record_info(fields, ?schema)}
                                                     ]),
+    mnesia:wait_for_tables([?schema], infinity),
     load_static_config();
 init(copy) ->
     ?tp(debug, rlog_schema_init,
         #{ type => copy
          }),
-    ok = ekka_mnesia:copy_table(?schema, ram_copies).
+    ok = ekka_mnesia:copy_table(?schema, ram_copies),
+    mnesia:wait_for_tables([?schema], infinity),
```

Comment:
take a look at `ekka_mnesia:do_wait_for_tables` we have some method to prevent infinity wait when some node is offline while pulling the table from remote. I am unsure about if that applies here but I think it is no harm.